traveler_core_kit/src/main/java/com/guestlogix/travelercorekit/models/DateRangeFilter.java

```diff
@@ -1,27 +1,25 @@
 package com.guestlogix.travelercorekit.models;
 
-import androidx.annotation.Nullable;
-
 import java.io.Serializable;
 import java.util.Date;
 
 public class DateRangeFilter implements Serializable {
-    @Nullable
+
     private Date startDate;
-    @Nullable
     private Date endDate;
 
-    public DateRangeFilter(@Nullable Date startDate, @Nullable Date endDate) {
+    public DateRangeFilter(Date startDate, Date endDate) {
+        if (startDate == null || endDate == null) {
```

Comment:
Wouldn't it be better if you make the constructor expect a Range<Date> ? by passing a Range<Date> you wouldn't have to do this check and users would have compiler help I believe

Reply:
Yeah you're right. Changed and pushed